easybuild/easyconfigs/s/SciPy-bundle/scipy-1.11.1_vectorization_error.patch

```diff
@@ -0,0 +1,104 @@
+Fixes the vectorization bug. see https://github.com/scipy/scipy/issues/16792
```

Comment:
@PetrKralCZ It's useful to mention the error that this patch fixes (to make it easier to find)
```
ValueError: buffer source array is read-only
```

Keep in mind that this implies updating the checksum for the patch file accordingly